### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by Zumbalamambo** [barloccia/MASK-CNN-for-actions-recognition-#1]
can you please share the pretrained model?

**Comment by adelmassimo** [comment]
Sorry,
but we have not a pretrained model, we're building a finetuned model of the
net.
Looking forward if you have need more help!
Adel and Gabriele

Il giorno ven 27 apr 2018 alle ore 19:45 Zumbalamambo <
notifications@github.com> ha scritto:
